.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,21 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Do we want this `.idea` folder in the repo?

Reply:
Absolutely not, my mistake. I thought I had this removed from the commit. I'm on a new computer and lost my global git ignore.